nexus/external-api/src/lib.rs

```diff
@@ -3542,104 +3542,137 @@ pub trait NexusExternalApi {
-    ) -> Result<HttpResponseUpdatedNoContent, HttpError>;
+        path_params: Path<params::AlertReceiverSelector>,
+        params: TypedBody<params::AlertSubscriptionCreate>,
+    ) -> Result<HttpResponseCreated<views::AlertSubscriptionCreated>, HttpError>;
 
-    /// Delete webhook receiver
+    /// Remove alert receiver subscription
     #[endpoint {
         method = DELETE,
-        path = "/v1/webhooks/receivers/{receiver}",
-        tags = ["system/webhooks"],
+        path = "/v1/alert-receivers/{receiver}/subscriptions/{subscription}",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_delete(
+    async fn alert_receiver_subscription_remove(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::WebhookReceiverSelector>,
+        path_params: Path<params::AlertSubscriptionSelector>,
     ) -> Result<HttpResponseDeleted, HttpError>;
 
-    /// Add webhook receiver subscription
+    /// List delivery attempts to alert receiver
+    ///
+    /// Optional query parameters to this endpoint may be used to filter
+    /// deliveries by state. If none of the `failed`, `pending` or `delivered`
+    /// query parameters are present, all deliveries are returned. If one or
+    /// more of these parameters are provided, only those which are set to
+    /// "true" are included in the response.
+    #[endpoint {
+        method = GET,
+        path = "/v1/alert-deliveries",
+        tags = ["system/alerts"],
+    }]
+    async fn alert_delivery_list(
+        rqctx: RequestContext<Self::Context>,
+        receiver: Query<params::AlertReceiverSelector>,
+        state_filter: Query<params::AlertDeliveryStateFilter>,
+        pagination: Query<PaginatedByTimeAndId>,
+    ) -> Result<HttpResponseOk<ResultsPage<views::WebhookDelivery>>, HttpError>;
+
+    /// Request re-delivery of alert
     #[endpoint {
         method = POST,
-        path = "/v1/webhooks/receivers/{receiver}/subscriptions",
-        tags = ["system/webhooks"],
+        path = "/v1/alerts/{alert_id}/resend",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_subscription_add(
+    async fn alert_delivery_resend(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::WebhookReceiverSelector>,
-        params: TypedBody<params::WebhookSubscriptionCreate>,
-    ) -> Result<HttpResponseCreated<views::WebhookSubscriptionCreated>, HttpError>;
+        path_params: Path<params::AlertSelector>,
+        receiver: Query<params::AlertReceiverSelector>,
+    ) -> Result<HttpResponseCreated<views::AlertDeliveryId>, HttpError>;
+
+    // ALERTS: WEBHOOKS
 
-    /// Remove webhook receiver subscription
+    /// Create webhook receiver
     #[endpoint {
-        method = DELETE,
-        path = "/v1/webhooks/receivers/{receiver}/subscriptions/{subscription}",
-        tags = ["system/webhooks"],
+        method = POST,
+        path = "/v1/webhook-receivers",
```

Comment:
should `alert` be the top-level path?

Reply:
see my discussion of nesting in https://github.com/oxidecomputer/omicron/pull/8169#discussion_r2093516103